compiler/damlc/daml-doc/test/DA/Daml/GHC/Damldoc/Tests.hs

```diff
@@ -154,7 +154,7 @@ unitTests =
                                 check $ isNothing $ td_descr t
                                 f1 <- getSingle $ td_payload t
                                 check $ isNothing $ fd_descr f1
-                                ch <- getSingle $ td_choices t
+                                ch <- getSingle $ td_choicesWithoutArchive t
```

Comment:
@associahedron Are you ok with changing it like this?

Reply:
This is fine, but perhaps the filter should be done when calculating `td_choices` instead? I feel like we don't generally want to show the `External:Archive` choice anyway.

Reply:
Opinions on whether we should show the `Archive` choice in the generated docs are a bit split. They show up in the transactions, e.g., in the scenario view in VS Code. But in some sense they are also an implementation detail covering up that we got our basic building blocks in the ledger model slightly wrong.
@bame-da You wanna weigh in?

Reply:
I think that until we make `Archive` its own node in the transaction graph, we should treat it just like any other choice, with the single exception that it's added to every template implicitly.

Reply:
:+1: I don't have a strong opinion either way. The damldoc changes here are fine.